src/js/base/ripe.js

```diff
@@ -1380,7 +1384,7 @@ ripe.Ripe.prototype.update = async function(state = null, options = {}, children
 
     try {
         this.updatePromise = _update();
-        if (options.noAwaitLayout) return true;
+        if (options.noAwaitLayout || this.remoteInitialsBuilderLogic) return true;

```

Comment:
Why this? Shouldn't the flag just control if the `initialsBuilder` is called and the `logic` param in the compose URL?

Reply:
Unfortunately it wasn't that simple. The problem is: by default we have an await for the last update logic which means that if you do 2 requests one after the other (example "1" and after "12") the second one will await the first one. We don't want that behavior when we `remoteInitialsBuilderLogic` is set to true as it would create that problem of rendering the "1" and then only the "12" as the cancelling logic is in the initials builder promise which is not used here.

Reply:
This this update is also can include the configurator. Won't this break any behaviour with that?

Reply:
Note that this doesn't alter the default behavior of the SDK - it's not a breaking change.
Only if you set the option `remoteInitialsBuilderLogic` to true will the SDK behavior differently and not wait the first update to be complete before processing the second one.

Also, I'm not sure how it would break but anyway I tested this customizing a swear vyner with the `remoteInitialsBuilderLogic` set to true and everything works. In fact, I found it to be even faster when changing parts but are you thinking a specific scenario?

Reply:
What I'm saying is that this is a specific logic for image, so it should not imply changes for the overall behaviour of SDK but only in its image. For example, it makes sense to not await when the variable `noAwaitLayout` is set (it's right in the name), but it is weird that the configuration for remote iniitals builder changes that logic (when it is only used in image)
One possible problem might appear using the preferred frame logic in the configurator (can be enabled in the White technical modal) and switching parts fast.

Reply:
@3rdvision I know this is p-high, so if you want you can move this forward and then you talk better about this with João

Reply:
I added `noAwaitLayout` as an option.
In that scenario, by default the SDK would have `noAwaitLayout` set to false and that way it wouldn't happen.